tutorials/README.md

```diff
@@ -0,0 +1,26 @@
+
+# RabbitMQ C AMQP tutorials
+
+## Introduction
+
+This directory holds example tutorial code that implement the six tutorials 
+found at: 
+
+ - <https://www.rabbitmq.com/getstarted.html>
+
+#### Prereqs:
+- PHP5.6.x
+- librabbitmq 0.5.x
```

Comment:
0.5.x is too old, maybe it worth mention that latest one required? And if you have php-amqp extension as a requirement below, then if one install it it will already have librabbitmq. So this requirement may not be obvious for newcomers.